Sources/SwiftDocC/LinkTargets/LinkDestinationSummary.swift

```diff
@@ -254,23 +258,94 @@ extension LinkDestinationSummary {
+            // Only symbol documentation currently support multi-language variants (rdar://86580915)
+            self.init(
+                kind: documentationNode.kind,
+                language: documentationNode.sourceLanguage,
+                path: path,
+                referenceURL: referenceURL,
+                title: ReferenceResolver.title(forNode: documentationNode),
+                abstract: (documentationNode.semantic as? Abstracted)?.renderedAbstract(using: &compiler),
+                availableLanguages: documentationNode.availableSourceLanguages,
+                platforms: platforms,
+                taskGroups: taskGroups,
+                usr: nil,
+                declarationFragments: nil,
+                redirects: redirects,
+                variants: []
+            )
+            return
+        }
+        
+        // Precompute the summarized elements information so that variants can compare their information against it and remove redundant duplicate information.
+        
+        // Multi-language symbols need to access the default content via the variant accessors (rdar://86580516)
+        let kind = DocumentationNode.kind(forKind: (symbol.kindVariants[summaryTrait] ?? symbol.kind).identifier)
+        let title = symbol.titleVariants[summaryTrait] ?? symbol.title
+        
+        func renderSymbolAbstract(_ symbolAbstract: Paragraph?) -> Abstract? {
+            guard let abstractParagraph = symbolAbstract, case RenderBlockContent.paragraph(let inlineContent)? = compiler.visitParagraph(abstractParagraph).first else {
+                return nil
+            }
+            return inlineContent
+        }
+        
+        let abstract = renderSymbolAbstract(symbol.abstractVariants[summaryTrait] ?? symbol.abstract)
+        let usr = symbol.externalIDVariants[summaryTrait] ?? symbol.externalID
+        let declaration = (symbol.subHeadingVariants[summaryTrait] ?? symbol.subHeading).map { subHeading in
+            subHeading.map { DeclarationRenderSection.Token(fragment: $0, identifier: nil) }
+        }
+        let language = documentationNode.sourceLanguage
+        
+        let variants: [Variant] = documentationNode.availableVariantTraits.compactMap { trait in
+            // Skip the variant for the summarized elements source language.
+            guard let interfaceLanguage = trait.interfaceLanguage, interfaceLanguage != documentationNode.sourceLanguage.id else {
+                return nil
+            }
+            
+            let declarationVariant = symbol.subHeadingVariants[trait].map { subHeading in
+                subHeading.map { DeclarationRenderSection.Token(fragment: $0, identifier: nil) }
+            }
+            
+            let abstractVariant: Abstract?? = symbol.abstractVariants[trait].map { renderSymbolAbstract($0) }
```

Comment:
Should or can we use the bespoke typealias you made for this?

Reply:
Updated